### ISSUE TITLE
code problem

### ISSUE THREAD

**Issue by JuliusMa** [NaJaeMin92/CoVi#3]
swap_src_labels = src_labels[mask_idx]
            pure_tgt_top1 = top1_label[mask_idx]
            swap_ul_loss, swap_lu_loss = 0, 0
            for i in range(num_of_mask):
                swap_ul_loss += sample_wise_loss(upper_out[i], swap_src_labels[i], top1_lower[i], upper_ratio[i])
                swap_lu_loss += sample_wise_loss(lower_out[i], pure_tgt_top1[i], top1_upper[i].cuda(), lower_ratio[i])
Is the swap_src_labels equal to the top1_lower? As the paper saying that there should be the top2 and top1 labels here, is the code correct?
